pypeit/spec2dobj.py

```diff
@@ -366,7 +366,8 @@ def build_primary_hdr(self, raw_header, spectrograph, master_key_dict=None, mast
         # TODO Should the spectrograph be written to the header?
         hdr['PIPELINE'] = str('PYPEIT')
         hdr['PYPELINE'] = spectrograph.pypeline
-        hdr['SPECTROG'] = spectrograph.spectrograph
+        # TODO: Why isn't this PYP_SPEC like it is everywhere else?
```

Comment:
No idea.  Let's use PYP_SPEC

Reply:
Changed.  I had to change this in `scripts/coadd_2dspec.py` and `scripts/coadd_datacube.py`, as well.  We'll see if that means I need to generate a new Cooked...